lib/dataset/include/scipp/dataset/map_view.h

```diff
@@ -44,6 +44,11 @@ auto slice_map(const Sizes &sizes, const T &map, const Slice &params) {
       if (value.dims()[params.dim()] == sizes[params.dim()]) {
         out[key] = value.slice(params);
       } else { // bin edge
+        if (params.stride() != 1)
+          throw except::SliceError(
+              "Object has bin-edges along dimension " +
+              to_string(params.dim()) + " so slicing with stride " +
+              std::to_string(params.stride()) + " != 1 is not valid.");
```

Comment:
Why? Is this a fundamental problem or just a limitation of our implementation?

Reply:
It is a fundamental problem from how we store bin edges. It would be possible if we stored, e.g., `(left, right)` for each bin, but not with storing edges as a normal array of length N+1.

Reply:
Gotcha